core/src/main/java/io/th0rgal/oraxen/bbmodel/BBModelTemplate.java

```diff
@@ -0,0 +1,26 @@
+package io.th0rgal.oraxen.bbmodel;
```

Comment:
Would probably want this under oraxen/pack/bbmodel as it doesnt really need to affect anything else